src/styles/components/_columns.scss

```diff
@@ -14,3 +14,43 @@
   // sass-lint:enable-all
   grid-template-columns: repeat(auto-fit, minmax(30ch, 1fr));
 }
+
+.w-columns .w-compare {
+  margin: 0;
+  /// adds the breakpoint styles
```

Comment:
I'd suggest removing the notes about adding breakpoint styles.

Reply:
OK. Was just following the convention in other style sheets.